### ISSUE TITLE
Small typo in the document of Module cellosaurus

### ISSUE THREAD

**Issue by evanswang** [biopython/biopython#1549]
### Setup

I am reporting a problem with Biopython version, Python version, and operating
system as follows:

```python
import sys; print(sys.version)
import platform; print(platform.python_implementation()); print(platform.platform())
import Bio; print(Bio.__version__)
```

``` python
Python 2.7.10 (default, Sep  3 2015, 10:11:44) 
[GCC 4.6.3] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys; print(sys.version)
2.7.10 (default, Sep  3 2015, 10:11:44) 
[GCC 4.6.3]
>>> import platform; print(platform.python_implementation()); print(platform.platform())
CPython
Linux-3.2.0-105-generic-x86_64-with-debian-wheezy-sid
>>> import Bio; print(Bio.__version__)
1.65
```

### Expected behaviour

The first example in the document (http://biopython.org/DIST/docs/api/Bio.ExPASy.cellosaurus-module.html) contains a typo.

```python
>>> from Bio.ExPASy import Cellosaurus
```
"Cellosaurus" should be "cellosaurus".

### Actual behaviour

```python
>>> from Bio.ExPASy import Cellosaurus
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name Cellosaurus
```

### Steps to reproduce

```python
>>> from Bio.ExPASy import Cellosaurus
```

**Comment by peterjc** [comment]
Thank you, the typo comes from here:
https://github.com/biopython/biopython/blob/master/Bio/ExPASy/cellosaurus.py#L21

Do you want to submit a pull request to fix this? You can do this entirely within the GitHub web-interface if you like.

**Comment by evanswang** [comment]
Hello,

I created a pull request #1550 to fix #1549. Please have a look. Thank you!

Best wishes,

Shicai Wang

**Comment by evanswang** [comment]
Yes, I agree to dual licence my Biopython contributions as per #898. Thank you! 

Best wishes,

Shicai
